plugins/deployment_manager/DeploymentManager.ts

```diff
@@ -38,15 +35,17 @@ export class DeploymentManager {
   _signers: SignerWithAddress[];
 
   constructor(
+    network: string,
     deployment: string,
     hre: HardhatRuntimeEnvironment,
     config: DeploymentManagerConfig = {}
   ) {
+    this.network = network;
     this.deployment = deployment;
     this.hre = hre;
     this.config = config;
 
-    this.cache = new Cache(deployment, config.writeCacheToDisk ?? false, config.baseDir);
+    this.cache = new Cache(`${this.network}/${this.deployment}`, config.writeCacheToDisk ?? false, config.baseDir);
```

Comment:
this is slightly strange; previously you could derive the path to the roots.json file from the "deployment" alone.

the other option would be to update `Cache` to take in both network and deployment.

Reply:
updated so that Cache takes a network and deployment